lib/portage/dbapi/bintree.py

```diff
@@ -575,7 +570,7 @@ def __init__(
             "SLOT": "0",
             "USE": "",
         }
-        self._pkgindex_inherited_keys = ["CHOST", "repository"]
+        self._pkgindex_inherited_keys = ["BINPKG_FORMAT", "CHOST", "repository"]
```

Comment:
This looks like an unintentional revert.

Reply:
Indeed, thanks! Fixed.